src/core/gltf2importer/meshparser.cpp

```diff
@@ -123,6 +125,201 @@ Qt3DRender::QAttribute *decodeAttribute(const draco::PointCloud *pointCould,
     return attribute;
 }
 #endif
+
+int vertexBaseTypeSize(Qt3DRender::QAttribute::VertexBaseType vertexBaseType)
+{
+    switch (vertexBaseType) {
+    case Qt3DRender::QAttribute::UnsignedByte:
+        return 1;
+    case Qt3DRender::QAttribute::UnsignedShort:
+        return 2;
+    case Qt3DRender::QAttribute::UnsignedInt:
+        return 4;
+    }
+}
+
+
+std::array<unsigned int, 3> findVertexIndicesInFace(const Qt3DRender::QAttribute *indexAttr, const int iFace) {
```

Comment:
Did you run clang-format? This curly brace should be on its own line.

Reply:
Also, dare I ask for unit tests? :-)

Reply:
Looks like clang-format was disabled